app/models/match_decisions/nine/nine_assign_case_contact.rb

```diff
@@ -89,6 +90,12 @@ def accessible_by? contact
         contact.in?(match.send(contact_actor_type))
     end
 
+    def step_decline_reasons(_contact)
+      [
```

Comment:
Why no `@step_decline_reasons =` here, but it is in some of the others like `MatchDecisions::Nine::NineAssignManager`?